.github/update_conda_lock.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
```

Comment:
This file needs a nice block comment.  I believe your PR comment has some of the required information, but it should be in the python file, not just the github PR.